src/components/Controls.svelte

```diff
@@ -1,22 +1,23 @@
+<link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@48,700,0,0" />
+
 <script lang="ts">
+	import { createEventDispatcher } from 'svelte';
+	import { construct_svelte_component } from 'svelte/internal';
 	import ControlStore from '../stores/ControlStore';
+	
 	// context vs store;
 	// can only play one song at a time; player controls was extracted
-
+	const dispatch = createEventDispatcher();	
 	export let songid: string; // required to make modifications to songid (ie add to playlist and more)
-	export let expanded: boolean; // true: card is expanded false: card is collapsed
-	//export let audio: Audio       is audio track for song, is an HTML Audio Element
+	export let audio: HTMLAudioElement;       //is audio track for song, is an HTML Audio Element
+	export let added: boolean; //whether or not song is added to playlist
+	export let liked: boolean; //whether or not song is favorited
+	export let position: number;
 	let status = {
 		PLAY: 'PLAY',
 		PAUSE: 'PAUSE'
 	};
 	let currStatus = status.PAUSE;
-	let trackPosition = 0;
-	//on expanded value change
-	if (expanded == false) {
-		trackPosition = 0;
-		currStatus = status.PAUSE;
-	}
 
 	//on currentSongId value change
 	if ($ControlStore.currentSongId != songid) {
```

Comment:
always use `!==` or `===` very bad practice to use otherwise. JavaScript has automatic type coercion and is great for simple apps, but as of right now, the complexities outweigh this benefit. Using coerced equality checkers undermines our purpose of using TypeScript (enforcing types).

The most important benefits is improve developer experience, and secondly, minimize potential areas for bugs to occur.

Reply:
whats the reasoning behind this not being changed?